src/gui/modals/SetupDialog.cpp

```diff
@@ -723,10 +724,18 @@ SetupDialog::SetupDialog(ConfigTabs tab_to_open) :
 		m_assignableMidiDevices->setCurrentIndex(current);
 	}
 
+	// MIDI Recording tab
+	auto* midiRecordingTab = new TabWidget(tr("Behavior when recording"), midi_w);
+	int mwCounter=0;
+	addLedCheckBox(tr("Auto-quantize notes in Piano Roll"), midiRecordingTab, mwCounter,
+		m_midiAutoQuantize, SLOT(toggleMidiAutoQuantization(bool)), false);
```

Comment:
I wonder if you could use a function pointer here instead of the ``SLOT`` macro?

Reply:
The addLedCheckbox convenience function takes a `const char*` resolved SLOT there, so a std::function / lambda would make this more messy. However, this is in conflict with #6779, which removes the LedCheckboxes from this dialog, so I could adapt this part in preparation for that merge.

Reply:
Just tested it, yeah. It could work, but you would probably have to change the type from ``const char*`` to a function pointer type, which may break the other ones.